Makefile

```diff
@@ -50,6 +50,3 @@ clean:
 	find . -name '*.pyc' -exec rm -f {} +
 	find . -name '*.pyo' -exec rm -f {} +
 	find . -type d -name __pycache__ -exec rm -r {} \+
-
-dev: 
-	pip install -q bump2version
```

Comment:
This is because bump2version isn't used as part of a dev environment?

Reply:
I just moved it to `requirements-dev.txt` to be consistent with what we do in other repos.

Reply:
Unless `make dev` is a thing we do in other places, then I'm happy to adjust that.

Reply:
No, cool. Makes sense